cfme/tests/intelligence/test_chargeback.py

```diff
@@ -68,42 +67,49 @@ def chargeback_storage_rate(rate_type='fixed'):
         })
 
 
-def test_add_new_compute_chargeback(chargeback_compute_rate):
-    chargeback_compute_rate.create()
-    view = chargeback_compute_rate.create_view(
-        navigator.get_class(chargeback_compute_rate, 'All').VIEW)
-    view.flash.assert_success_message('Chargeback Rate "{}" was added'
-                                      .format(chargeback_compute_rate.description))
-    assert chargeback_compute_rate.exists
+@pytest.mark.tier(3)
+def test_compute_chargeback_duplicate_disallowed(request):
+    cb_rate = chargeback_compute_rate('fixed')
 
+    @request.addfinalizer
+    def _cleanup():
+        if cb_rate.exists:
+            cb_rate.delete()
 
-@pytest.mark.tier(3)
-def test_compute_chargeback_duplicate_disallowed(chargeback_compute_rate):
-    chargeback_compute_rate.create()
-    assert chargeback_compute_rate.exists
+    cb_rate.create()
+    assert cb_rate.exists
     with pytest.raises(AssertionError):
-        chargeback_compute_rate.create()
+        cb_rate.create()
     # view should still be on the add form
-    view = chargeback_compute_rate.create_view(
-        navigator.get_class(chargeback_compute_rate, 'Add').VIEW)
+    view = cb_rate.create_view(
+        navigator.get_class(cb_rate, 'Add').VIEW)
     view.flash.assert_message('Description has already been taken', t='error')
     # cancel form, check all redirect
     view.cancel_button.click()
-    view = chargeback_compute_rate.create_view(
-        navigator.get_class(chargeback_compute_rate, 'All').VIEW)
+    view = cb_rate.create_view(
+        navigator.get_class(cb_rate, 'All').VIEW)
```

Comment:
Ah, these still don't have collection classes.

Reply:
Collection class needs to be added to this file as well :https://github.com/ManageIQ/integration_tests/blob/master/cfme/intelligence/chargeback/rates.py#L136

I could submit a follow up PR to add collection class to rates.py and test_chargeback.py .